app/actions/remote/entry/common.ts

```diff
@@ -260,13 +261,15 @@ export async function deferredAppEntryActions(
     initialTeamId?: string, initialChannelId?: string) {
     // defer sidebar DM & GM profiles
     let channelsToFetchProfiles: Set<Channel>|undefined;
-    if (chData?.channels?.length && chData.memberships?.length) {
-        const directChannels = chData.channels.filter(isDMorGM);
-        channelsToFetchProfiles = new Set<Channel>(directChannels);
+    setTimeout(async () => {
```

Comment:
By delaying this `fetchPostsForUnreadChannels`, the home screen gets time to load and it properly catches the `FETCHING_POSTS` event.

Reply:
Agreed.  I'm going to merge this PR for the time being.